drivers/dma/Kconfig.nxp_sof_host_dma

```diff
@@ -0,0 +1,34 @@
+# Copyright 2023 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+config DMA_NXP_SOF_HOST_DMA
+	bool "NXP DMA driver used by SOF's host component"
+	default y
+	depends on DT_HAS_NXP_SOF_HOST_DMA_ENABLED
+	help
+		Enable NXP's DMA driver used by
+		SOF (Sound Open Firmware) host
+		component. Specifically, this driver
+		is used by the SOF host component to
+		perform transfers between the host
+		memory and firmware (local) memory, which
+		can be accessed without an actual
+		DMA engine.
+
+if DMA_NXP_SOF_HOST_DMA
+
+config DMA_NXP_SOF_HOST_DMA_ALIGN
+	int "Alignment (in bytes) required for memory regions passed to this driver"
+	default 8
+	help
+		Use this to set the alignment (in bytes)
+		which shall be used by entities employing
+		this driver to adjust a memory region's size
+		and base address. Since this driver doesn't
+		actually have any hardware to back it up this
+		configuration doesn't make much sense as there's
```

Comment:
Is this config required by DMA engine API?
If not, and we're not using it, can we just set a `const` value, like 8, directly here: https://github.com/zephyrproject-rtos/zephyr/pull/65281/files#diff-6d7e1df734af6a9e72ede3596b4e7c67407cb61a94c2a8efbc5421cb7ef047f9R239 ?

Reply:
The value here was chosen for ARM64. Ideally, I plan on going for a 4B alignment on XTENSA, so rather than doing this with a macro-based design I figured it would be better to change the alignment through a config. This also gives you the freedom to play around with the alignment.

Of course, if you still think it's not necessary we can just remove it. The constant-based approach works fine with the current test setup I have.

Reply:
Ok, so is specific to NXP, but might be different per platform: imx8m, imx8x, imx8 (on xtensa), or imx93 (on arm64)?
If this is the case, I understand the point of this config - is easier through a config than a switch in the DMA driver.

Reply:
I'd say architecture rather than platform

Reply:
Ok, thanks. Now I understand and agree with this approach.